script/blank-site

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+#
+# Runs the `jekyll new` command with switch `--blank` and builds the blank site as a sanity check.
+
+set -e
+
+print_label() {
+  echo -e "\e[33m$0:\e[0m \e[36m$1..\e[0m"
+}
+
+# --
+
+print_label "Setting up 'tmp' directory"
+mkdir -p ./tmp
+rm -Rf ./tmp/default-site
+
+print_label "Creating new BLANK site"
+rm -Rf ./tmp/blank-site
+bundle exec jekyll new tmp/blank-site --blank
```

Comment:
I'm hesitant to duplicate script/default-site like this... I'm wondering if we can either reuse script/default-site for 99% of this script, or, better yet, use a Cucumber test for this.

Reply:
I think Cucumber is my favorite option here an alternative is to re-use default-site: #9259.